ledger/acctonline.go

```diff
@@ -150,6 +153,11 @@ func (ao *onlineAccounts) initializeFromDisk(l ledgerForTracker, lastBalancesRou
 			return err0
 		}
 
+		ao.onlineRoundParamsData, err0 = accountsOnlineRoundParams(tx)
```

Comment:
I think we can use `ao.onlineRoundParamsData` as combined round deltas + cache (the current usage is inconsistent: if used as a delta, we should not load data into deltas, if used as a cache, we should not remove what is committed).
The idea:
 - load all data as it is done here
 - add data into it in newBlock (also done)
 - keep track of what is already committed, what is the latest element to keep
 - slice it when to have only [Lastest - MaxBalLookback, Lastest] elements
 - implement a new `roundParamsOffset` function (`roundOffset` cannot be used) to map a requested round number to the position within `ao.onlineRoundParamsData`.
 - Be aware of when to modifying. It must be done under a lock, and the lookup code should be aware about the fact `ao.onlineRoundParamsData` changes.